static/js/poll_widget.js

```diff
@@ -192,6 +200,7 @@ export function activate({
         is_my_poll,
         question,
         options,
+        render_user_ids_function: people.safe_full_names,
```

Comment:
Can we pick a better name for this?  I think `render_safe_names_for_users` or something might be better.  `render_user_ids` sounds like it's rendering user_ids.

Reply:
Done.

Reply:
`render_safe_` makes it sound like there's HTML escaping going on, which as far as I can see is not the case.